bitnami/mastodon/values.yaml

```diff
@@ -104,62 +104,25 @@ image:
 environment: production
 ## @param adminUser Mastodon admin username
 ##
-adminUser: user
+adminUser: ""
 ## @param adminEmail Mastodon admin email
 ##
-adminEmail: user@changeme.com
+adminEmail: ""
```

Comment:
Any specific reason for removing `adminEmail` and `adminUser`? That would impact the container implementation https://github.com/bitnami/containers/blob/main/bitnami/mastodon/4/debian-11/rootfs/opt/bitnami/scripts/libmastodon.sh#L241.

Reply:
Thanks a lot for explaining it!